amoro-format-iceberg/src/main/java/org/apache/amoro/table/TableProperties.java

```diff
@@ -42,6 +42,11 @@ private TableProperties() {}
   public static final String TABLE_CREATE_TIME = "table.create-timestamp";
   public static final long TABLE_CREATE_TIME_DEFAULT = 0L;
 
+  /** table comment related properties */
+  public static final String COMMENT = "comment";
+
+  public static final String COMMENT_DEFAULT = "";
```

Comment:
I suggest that don't need a default value, and that it would be more user-friendly if the page doesn't show anything without comment. WDYT?